src/ssl.c

```diff
@@ -38142,6 +35830,10 @@ int wolfSSL_i2d_ECDSA_SIG(const WOLFSSL_ECDSA_SIG *sig, unsigned char **pp)
 }
 /* End ECDSA_SIG */
 
+
+/* Remove this prototype when code is reorganized. */
+static int wolfSSL_RAND_Init(void);
```

Comment:
This is already at line 4983. We should only have one prototype.

Reply:
That got me too:
4983: static int wolfSSL_RAND_InitMutex(void);
35835: static int wolfSSL_RAND_Init(void);